test/test.js

```diff
@@ -386,8 +386,8 @@ describe("properties", function () {
       features: [{
         type: "Feature",
         properties: {
-          tags: { name: "name" },
-          name: "not_name"
+          tags: { id: "name" },
+          ref: "not_name"
```

Comment:
This reflects a change in property precedence when configuring a name.
Before this pull request, the options were, in order of decreasing precedence:
1. inferred from tags (`tags.name`, `tags.id`, `tags.ref`)
2. inferred from bare properties (`name`, `id`, `ref`)

After this pull request, the options would be, in order of decreasing precedence:
1. explicit in a bare property (`name`)
2. inferred from tags (`tags.name`, `tags.id`, `tags.ref`)
3. inferred from bare properties (`name`, `id`, `ref`)

Note that the occurrence of `name` in the third option is superfluous.

Because of this change in precedence, I suggest that the next version would be `0.6.0`, if this PR is accepted.